### PR TITLE
Adding Sieve_Of_Erantosthenes in kotlin

### LINKED ISSUE #1188: Sieve of Eratosthenes in kotlin
Please Assign it to me. @jainaman224

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1188

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding Sieve Of Erantosthenes in kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Sieve_Of_Erantosthenes.kt

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        if (prime[i]) {
+            for (j in (2 * i)..(num) step i) {
+                prime[j] = false
+            }
+        }
+    }
+    // Printing Prime Numbers In the range of (1 to given Number)
+    for (primes in 2 until num) {
+        if (prime[primes]) {
+            print(" $primes")
```
Comment (marwahmanbir) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt]: Fix spacing around ``$primes`` and add an extra line at the end of file. Rest looks good. :+1:

**Comment by marwahmanbir** [comment]
Remove space after # where you've mentioned the issue number.

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt (hunk 1 above)
Comment (shubhammantri1) [reply Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt]: Done

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        if (prime[i]) {
+            for (j in (2 * i)..(num) step i) {
+                prime[j] = false
+            }
+        }
+    }
+    // Printing Prime Numbers In the range of (1 to given Number)
+    for (primes in 2 until num) {
+        if (prime[primes]) {
+            print("$primes ")
```
Comment (marwahmanbir) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt]: Please remove the space after ``$primes``.

Comment (shubhammantri1) [reply Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.kt]: But then output is changed and all numbers in the output are merged with each other??

**Pull request merged**